libs/extensions/angular/image-banner/src/image-banner.component.scss

```diff
@@ -0,0 +1,171 @@
+@use '@kirbydesign/core/src/scss/utils';
+@use '@kirbydesign/core/src/scss/interaction-state';
+
+$blur-radius: 60px;
+$max-content-width: 324px;
```

Comment:
Is this just a magic number or can it be calculated somehow to give more clarity?

Reply:
Unfortunately just a magic number specific to this component :/